CONTRIBUTING.md

```diff
@@ -42,6 +42,8 @@ git clone git@github.com:meetalva/alva.git
 npm i && npm start
 ```
 
+(Note: Please use *npm* only. We dropped the support for Yarn for now, as we wired our release processed to the package-lock.json file.)
```

Comment:
Shouldn’t it be "process" instead of "processed"?

Reply:
Yes. I fixed that.